tests/kafkatest/tests/streams/streams_broker_down_resilience_test.py

```diff
@@ -15,10 +15,10 @@
 
 import time
 from kafkatest.services.streams import StreamsBrokerDownResilienceService
-from kafkatest.tests.streams.base_streams_test import BaseStreamsTest
+from kafkatest.services.streams import StreamsTestBaseService
 
 
-class StreamsBrokerDownResilience(BaseStreamsTest):
+class StreamsBrokerDownResilience(StreamsTestBaseService):
```

Comment:
Here you switched the parent class from `BaseStreamsTest` to `StreamsTestBaseService`.  That is fine, but this test relies on methods defined in `BaseStreamsTest` that will need to get added to `StreamsBrokerDownResilience`.

1. `assert_produce_consume` which itself delegates to other methods in `BaseStreamsTest`
2. There are configs specific to this test where we don't want the default Streams settings https://github.com/apache/kafka/blob/trunk/tests/kafkatest/tests/streams/base_streams_test.py#L83-L86. But we also don't want to put these in `StreamsTestBaseService` as we also don't want these configs for all Streams system tests either.

Overall it might be easier to keep the parent class for `StreamsBrokerDownResilience` to `BaseStreamsTest`, and change the parent of `BaseStreamsTest` to `StreamsTestBaseService` and override the configs in `StreamsBrokerDownResilience.prop_file` like you have (including the ones mentioned above) done already. But I'll leave it up to you to decide.

Reply:
Thank you for pointing this out @bbejeck. As for the decision on my end, I'll follow your suggestion as it does appear to be reasonable, and I am not yet familiar with the entire code base.